packages/internal/src/generate/plugins/rw-typescript-resolvers/index.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  Types,
+  PluginFunction,
+  getCachedDocumentNodeFromSchema,
+  oldVisit,
+} from '@graphql-codegen/plugin-helpers'
+import {
+  TypeScriptResolversPluginConfig,
+  plugin as basePlugin,
+} from '@graphql-codegen/typescript-resolvers'
+import { GraphQLSchema } from 'graphql'
+
+import { RwTypeScriptResolversVisitor } from './visitor.js'
+
+export const plugin: PluginFunction<
+  TypeScriptResolversPluginConfig,
+  Types.ComplexPluginOutput
+> = (
+  schema: GraphQLSchema,
+  _documents: Types.DocumentFile[],
+  config: TypeScriptResolversPluginConfig
+) => {
+  // This is the key change compared to the standard typescript-resolver
+  // plugin implementation - we use our own Visitor here.
+  // There are more changes done to this file, but they're all pretty much
```

Comment:
Could you expand a little bit on what's being removed here please?

Reply:
It might be easiest if you just look at the original file here: https://github.com/dotansimha/graphql-code-generator/blob/c6c60a3078f3797af435c3852220d8898964031d/packages/plugins/typescript/resolvers/src/index.ts

But as an example I removed everything wrapped by `if (visitor.hasFederation())` because that was `false` for us. Another example is I removed everything that has to do with schema stitching, because we don't do that.

Reply:
I think the changes we made together means that this comment (and mine) are outdated